prow/repoowners/repoowners.go

```diff
@@ -776,7 +777,7 @@ func (o *RepoOwners) FindReviewersOwnersForFile(path string) string {
 // FindLabelsForFile returns a set of labels which should be applied to PRs
 // modifying files under the given path.
 func (o *RepoOwners) FindLabelsForFile(path string) sets.String {
-	return o.entriesForFile(path, o.labels, false)
+	return o.entriesForFile(path, o.labels, false).Set()
```

Comment:
@alvaroaleman here too, how to I "cast" a multisets.Set interface into a sets.String ?

Reply:
Yeah sorry, I didn't think properly when commenting, this wont work

Reply:
Ok, I revert the second commit...